code/modules/antagonists/clock_cult/clockwork_turfs.dm

```diff
@@ -258,12 +258,12 @@
 	if(istype(M) && is_servant_of_ratvar(M))
 		if(!LAZYLEN(heal_people))
 			START_PROCESSING(SSprocessing, src)
-		heal_people += M
+		LAZYADD(heal_people, M)
```

Comment:
should this also be lazyinited?

Reply:
No, lazyadd inits the list if its not already.

Reply:
https://github.com/BeeStation/BeeStation-Hornet/blob/5b5f7ee891822f852ecf0812086cfe759d2e2ed8/code/modules/antagonists/clock_cult/clockwork_turfs.dm#L248

If it's not lazyinited it's not lazy